config/locales/fr.yml

```
@@ -186,32 +186,12 @@ fr:
           still_in_charge: Oui, je suis toujours en charge
           will_change_soon: Oui, je suis toujours en charge mais pourrais passer la main d'ici quelques mois
           not_in_charge: Non, je ne suis plus en charge de cette habilitation %{api}
+        not_in_charge_alert:
+          text: Vous devez transférer vos habilitations sur le site de <a href="https://%{api}.v2.datapass.api.gouv.fr" target="_blank">DataPass</a>
```

Ok, j'aurai juste était un peu plus précis.

Proposition : 

```suggestion
          text: Si vous n'êtes plus en charge de cette habilitation, vous devez la transférer à une autre personne de votre organisation depuis le site de <a href="https://%{api}.v2.datapass.api.gouv.fr" target="_blank">DataPass</a>.<br/><br/>Cette démarche prend 2 minutes. La nouvelle personne en charge devra alors effectuer la prolongation du jeton.
```